src/main.js

```diff
@@ -8,29 +8,27 @@ var ideaFormSection = document.querySelector('.idea-form');
 var cardTitle = document.querySelector('.card-title');
 var bodyText = document.querySelector('.body-text');
 var ideaCardSection = document.querySelector('.idea-cards');
-var starFavButton = document.querySelector('.header-star');
 var ideaArray = [];
 
 window.addEventListener('keyup', formValidation);
-saveButton.addEventListener('click', createIdeaObject);
-ideaCardSection.addEventListener('click', deleteCard);
+window.addEventListener('click', clickHandler);
+// saveButton.addEventListener('click', createIdeaObject);
+// ideaCardSection.addEventListener('click', deleteCard);
 // ideaCardSection.addEventListener('click', starFavorite);
 
-// window.addEventListener('click', clickHandler);
 
 
-// function clickHandler(target) {
-//   if (event.target === saveButton) {
-//     createIdeaObject()
-//   }
-//   if (event.target === starFavButton) {
-//     starFavorite()
-//   }
-//   if (event.target === deleteCardButton) {
-//     starFavorite();
-//     deleteCard();
-//   }
-// }
+function clickHandler(event) {
+  if (event.target === saveButton) {
+    createIdeaObject(event);
+  }
+  if (event.target.classList.contains("star")) {
```

Comment:
Oh! That was brilliant! And I learned something new ... had no idea you could use contains with classList. Great to know!